xarray/tests/test_plot.py

```diff
@@ -3220,3 +3221,39 @@ def test_facetgrid_axes_raises_deprecation_warning() -> None:
             ds = xr.tutorial.scatter_example_dataset()
             g = ds.plot.scatter(x="A", y="B", col="x")
             g.axes
+
+
+@requires_matplotlib
+def test_plot1d_default_rcparams() -> None:
+    import matplotlib as mpl
+
+    ds = xr.tutorial.scatter_example_dataset(seed=42)
+
+    with figure_context():
+        # scatter markers should by default have white edgecolor to better
+        # see overlapping markers:
+        fig, ax = plt.subplots(1, 1)
+        ds.plot.scatter(x="A", y="B", marker="o", ax=ax)
+        np.testing.assert_allclose(
+            ax.collections[0].get_edgecolor(), mpl.colors.to_rgba_array("w")
+        )
+
+        # Facetgrids should have the default value as well:
+        fg = ds.plot.scatter(x="A", y="B", col="x", marker="o")
+        ax = fg.axs.ravel()[0]
+        np.testing.assert_allclose(
+            ax.collections[0].get_edgecolor(), mpl.colors.to_rgba_array("w")
+        )
+
+        # scatter should not emit any warnings when using unfilled markers:
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
```

Comment:
You could use

https://github.com/pydata/xarray/blob/3aa75c8d00a4a2d4acf10d80f76b937cadb666b7/xarray/tests/__init__.py#L180


```suggestion
        with assert_no_warnings():
```

(would still need the import)